src/main/java/practice/StreamPractice.java

```diff
@@ -23,7 +46,22 @@ public int findMinEvenNumber(List<String> numbers) {
      * But before that subtract 1 from each element on an odd position (having the odd index).
      */
     public Double getOddNumsAverage(List<Integer> numbers) {
-        return 0D;
+        List<Integer> modifiedNumbers = IntStream.range(0, numbers.size())
```

Comment:
one stream is enough here
```suggestion
        return IntStream.range(0, numbers.size())
```